src/Microsoft.ML.Data/DataLoadSave/TransformerChain.cs

```diff
@@ -128,6 +128,25 @@ public DataViewSchema GetOutputSchema(DataViewSchema inputSchema)
             return s;
         }
 
+        [BestFriend]
+        internal TransformerChain<ITransformer> RewindToLastPredictionTransformer()
+        {
+            int lastPredictorIndex = 0;
```

Comment:
>lastPredictorIndex [](start = 16, length = 18)

nit: technically, this isn't the index of the last predictor, it's the index+1... Might be less confusing to rename.